build_web_compilers/build.yaml

```diff
@@ -68,6 +81,15 @@ builders:
         compiler: dart2js
     applies_builders:
       - build_web_compilers|dart2js_archive_extractor
+  _stack_trace_mapper_copy:
```

Comment:
This is going to pollute generated build scripts for everyone. Not a huge impact but I wonder if we should avoid it?

Should we make "private" builders a first class concept?

Should we create a separate package for this in the mono_repo which is a `dev_dependency` instead?

Reply:
It doesn't actually pollute them - the builder is not even importable outside this project.

Reply:
Ah I forgot about that setup.